src/components/WOITopNavbar/index.tsx

```diff
@@ -114,21 +116,29 @@ const WOITopNavbar = (props: TopNavbarProps) => {
         border-color: ${borderColor};
         border-style: solid;
         border-width: ${borderThickness};
+        font-family: ${fontFamily ? fontFamily : 'Roboto'};
+        font-size: ${fontSize ? fontSize + 'px' : '16px'};
+        font-style: normal;
+        font-weight: ${fontWeight ? fontWeight : '400'};
+        line-height: normal;
+        color: ${textColor};
```

Comment:
What if textColor is not passed in the props? Is it a required prop? Or should a default textColor value be given as in case of fontWeight for example.

Reply:
its not a required prop, so even if it goes null, the default text color will be black: the default of css